packages/react-core/src/components/TimePicker/examples/TimePicker.md

```diff
@@ -0,0 +1,45 @@
+---
+id: Time picker
+section: components
+cssPrefix: pf-c-time-picker
+propComponents: ['TimePicker']
+beta: true
+---
+
+import { TimePicker } from '@patternfly/react-core';
+
+## Examples
+### Basic 12 hour
+```js
+import React from 'react';
+import { TimePicker } from '@patternfly/react-datetime';
```

Comment:
Replace `@patternfly/react-datetime` with `@patternfly/react-core` in all examples.